### ISSUE TITLE
Object.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by sholtomaud** [olmokramer/atom-stylus-autocompile#2]
Requiring `View` from `atom` is no longer supported.
Please require `atom-space-pen-views` instead:
  `{View} = require 'atom-space-pen-views'`
Add `"atom-space-pen-views": "^2.0.3"` to your package dependencies.

```
Object.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app.asar/exports/atom.js:66:11)
Object.<anonymous> (/Users/chr/.atom/packages/stylus-autocompile/lib/stylus-autocompile-view.coffee:1:1)
Object.<anonymous> (/Users/chr/.atom/packages/stylus-autocompile/lib/stylus-autocompile-view.coffee:221:4)
Module._compile (module.js:452:26)
```

**Comment by olmokramer** [comment]
This is the same as #1. Closing as duplicate.

I'll try to get to it, I think I can do it this weekend

**Comment by olmokramer** [comment]
I just released `v0.2.0` that should remove all the deprecation warnings.
